specs/reporters/xunit.cpp

```diff
@@ -12,7 +12,8 @@ go_bandit([]() {
 
     before_each([&]() {
       stm.str(std::string());
-      reporter.reset(new reporter::xunit(stm, formatter));
+      reporter.reset(new reporter::xunit(stm, formatter, true));
+      // TODO: do test without timing
```

Comment:
Please do, or don't (should contain `time="0` in both cases), but please don't add a TODO ;)